clang/test/CodeGen/X86/x86-atomic-float.c

```diff
@@ -0,0 +1,64 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 4
+// RUN: %clang_cc1 -triple x86_64-linux-gnu -target-cpu core2 %s -S -emit-llvm -o - | FileCheck %s
```

Comment:
Please can you add the i686 test coverage back? I mean that in most cases you should be able to share the check-prefixes

Reply:
Yes I added the test, I misinterpreted the earlier review.
Thanks!